onedal/datatypes/utils/common.hpp

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+* Copyright 2024 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <unordered_map>
+
+namespace oneapi::dal::python {
+
+template <typename Key, typename Value>
+inline auto inverse_map(const std::unordered_map<Key, Value>& input)
```

Comment:
Where is this used?

Reply:
It is also some debug code, that was not deleted. originally taken from #1568 but not used.